src/commands/patches.ts

```diff
@@ -365,7 +368,7 @@ async function createDraft(
 
 	const create: CreateDraft = { changes: [change] };
 
-	const commit = await container.git.getCommit(repository.uri, to);
+	const commit = args.commit ?? (await container.git.getCommit(repository.uri, to));
```

Comment:
Why is the `to` argument with the lookup not enough here?

Reply:
That's what I'm wondering as well. Does `container.git.getCommit` fail if the ref is to a stash? If so, should that be improved instead, or is there a `getStashCommit` option we could use?

Just trying to determine why the new `commit` arg would not be considered redundant.